Plugins/Public/playercntl_plugin/Message.cpp

```diff
@@ -1241,6 +1245,34 @@ namespace Message
 		return true;
 	}
 
+	bool Message::UserCmd_SetDeathTime(uint iClientID, const wstring &wscCmd, const wstring &wscParam, const wchar_t *usage)
+	{
+		wstring wscParam1 = ToLower(GetParam(wscParam, ' ', 0));
+		bool bShowChatDieTime = false;
+		if (!wscParam1.compare(L"on"))
+			bShowChatDieTime = true;
+		else if (!wscParam1.compare(L"off"))
+			bShowChatDieTime = false;
+		else
+		{
+			PrintUserCmdText(iClientID, L"ERR Invalid parameters");
+			PrintUserCmdText(iClientID, usage);
```

Comment:
return?

Reply:
Whoops. I suppose we don't want to turn it off if invalid parameters were used. 
Fixed.